src/amo/components/Collection/index.js

```diff
@@ -141,9 +141,12 @@ export class CollectionBase extends React.Component<Props> {
           <h1 className="Collection-title">
             {collection ? collection.name : <LoadingText />}
           </h1>
-          <p className="Collection-description">
-            {collection ? collection.description : <LoadingText />}
-          </p>
+          {collection ? (
+            <p
```

Comment:
I think it would be nicer if you changed this to `<span dangerouslySetInnerHTML={sanitizeHTML(collection.description)} />`.

That would allow you to keep wrapping it in a common container:

```js
<p className="Collection-description">
  {collection ? <span dangerouslySetInnerHTML={...} /> : <LoadingText />}
</p>
```

Reply:
Ah, that's a good call. It increases the output of the HTML a bit but it's certainly nicer to write.